tests/template_tests/test_partials.py

```diff
@@ -0,0 +1,303 @@
+import os
+from textwrap import dedent
+from types import ModuleType
+from unittest import mock
+
+from django.http import HttpResponse
+from django.template import TemplateDoesNotExist, TemplateSyntaxError, engines
+from django.template.backends.django import DjangoTemplates
+from django.template.loader import render_to_string
+from django.test import TestCase, override_settings
+from django.urls import path, reverse
+
+
+class PartialTagsTestCase(TestCase):
+    def test_partial_tags(self):
+        template = """
+        {% partialdef testing-partial %}HERE IS THE TEST CONTENT{% endpartialdef %}
+        {% partial testing-partial %}
+        """
+
+        engine = engines["django"]
+        t = engine.from_string(template)
```

Comment:
Given the repetition of this two lines, I think the test case could benefit from a property like:

```python
@property
def engine(self):
    return engines["django"]
```
And then, in every test:
```suggestion
        t = self.engine.from_string(template)
```